yarn-project/acir-simulator/src/avm/fixtures/index.ts

```diff
@@ -57,3 +57,25 @@ export function initExecutionEnvironment(overrides?: AvmExecutionEnvironmentOver
     overrides?.calldata ?? [],
   );
 }
+
+/**
+ * An interface that allows to override the default values of the GlobalVariables
+ */
+export interface GlobalVariablesOverrides {
+  chainId?: Fr;
+  version?: Fr;
+  blockNumber?: Fr;
+  timestamp?: Fr;
+}
+
+/**
+ * Create an empty instance of the Execution Environment where all values are zero, unless overriden in the overrides object
```

Comment:
@Maddiaa0 The comment "...of the Execution Environment ..." Is this a copy-paste leftover? We mean Global Variables instead, right?